content/07.supplemental.md

```diff
@@ -5,3 +5,5 @@
 ![Figure S2. Validation of Consensus SNV calls and Tumor Mutation Burden, Related to Figures 2 and 3.](images/supplement/Figure-S2.png){#fig:FigS2}
 
 ![Figure S3. Genomic instability of pediatric brain tumors, Related to Figures 2 and 3.](images/supplement/Figure-S3.png){#fig:FigS3}
+
+![Figure S4. Related to Figure 4. A, Pearson correlation for counts with FPKM from poly-A samples (R = 0.94, p < 2.2e-16, N = ). B, Pearson correlation for counts with FPKM from stranded samples (R = 0.97, p < 2.2e-16, N = ).](images/supplement/SuppTelomerase_Activities.png){#fig:FigS4}
```

Comment:
This caption does not reflect the updated content of Fig S4.

Reply:
updated this and will upload a new figure